infrastructure/server/waf-firewall.tf

```diff
@@ -0,0 +1,226 @@
+locals {
```

Comment:
I wonder if this file should be called waf-firewall-api-gateway, similar to how the waf-firewall-cloudfront is named?

Reply:
Hi Basil, my initially intention of the file is to have all WAF ACL rules defined in one file only, since they are just rules (not association) and since we don't have "modules" structure (such as "./modules/waf") in our Terraform file folder.
So the waf-firewall.tf file contains **both** the **api-gateway** rules and **cognito** rules to be in one location (and only split for Cloudfront due to limitation of our Terraform structure is server/frontend) .
However, if you prefer to split them I will do that.

Reply:
Thanks Basil. I will mark this one resolved.